package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "dita-bootstrap.lunr",
+  "description": "DITA Open Toolkit plug-in that extends HTML output from the DITA Bootstrap plug-in with a Lunr.js search box",
+  "homepage": "https://github.com/infotexture/dita-bootstrap.lunr",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/infotexture/dita-bootstrap.lunr.git"
+  },
+  "license": "Apache-2.0",
+  "scripts": {
+    "check": "npm run prettier-check",
+    "fmt": "npm run format",
+    "format": "prettier --write \"**/*\" --ignore-unknown",
+    "lint-staged": "lint-staged",
+    "prepare": "husky install",
+    "prettier": "prettier --write \"**/*\" --ignore-unknown",
+    "prettier-check": "prettier --check \"**/*\" --ignore-unknown",
+    "test": "prettier --list-different \"**/*\" --ignore-unknown"
+  },
+  "lint-staged": {
+    "**/*": "prettier --write --ignore-unknown"
+  },
+  "devDependencies": {
+    "@prettier/plugin-xml": "3.1.0",
```

Comment:
Add https://www.npmjs.com/package/lunr-languages and  https://www.npmjs.com/package/lunr as dependencies to  get dependabot to track for changes.

Reply:
Added in abcd6a8.